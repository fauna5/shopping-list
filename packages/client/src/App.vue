<template>
  <div id="app">
    <div class="conatiner">
      <div class="header">
        <img src="./assets/logo.svg"/>
        <h1>Shopping List</h1>
      </div>
      <item-input/>
      <list class="list"/>
    </div>
  </div>
</template>

<script>
import List from './components/List';
import ItemInput from './components/ItemInput';

export default {
  name: 'app',
  components: {
    ItemInput,
    List,
  },
};
</script>

<style scoped>
  .conatiner {
    max-width: 600px;
    margin: auto;
  }
  .header {
    height: 40px;
    background-color: #1795d4;
    color: #dff9ff;
    font-size: 12px;
    font-weight: normal;
    position: relative;
  }
  .header h1 {
    text-align: center;
    margin: 0px;
    padding-top: 7px;
  }
  .header img {
    height: 25px;
    position: absolute;
    top: 8px;
    left: 5px;
  }
</style>
