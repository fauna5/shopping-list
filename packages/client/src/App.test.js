import { shallowMount } from "@vue/test-utils";
import App from "./App";

describe("App", () => {
  it("should render", () => {
    const wrapper = shallowMount(App);
    expect(wrapper.element).toMatchSnapshot();
  });
});
