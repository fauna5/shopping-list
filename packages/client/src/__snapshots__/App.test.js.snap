// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render 1`] = `
<div
  id="app"
>
  <div
    class="conatiner"
  >
    <div
      class="header"
    >
      <img
        src="./assets/logo.svg"
      />
       
      <h1>
        Shopping List
      </h1>
    </div>
     
    <item-input-stub />
     
    <list-stub
      class="list"
    />
  </div>
</div>
`;
